### PR TITLE
ML_2_8_K-Nearest-Neighbour-Algorithm_(D).md

### LINKED ISSUE #2811: ML 2.8: K-Nearest Neighbour Algorithm (D)
### Description

Welcome to 'ML' Team, good to see you here

This issue will helps readers in acquiring all the knowledge that one needs to know about **K-Nearest Neighbour**.

To get assigned to this issue, add your **Batch Numbers** mentioned in the spreadsheet of "Machine Learning", this issue is only for **Documentation** content.

Domain : Machine Learning

Points to Note :

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.

- "Issue Title" and "PR Title should be the same. Include issue number along with it.

- Changes should be made inside the `Machine Learning/Machine_Learning/Supervised_Machine_Learning` Branch.

- Follow Contributing Guidelines & Code of Conduct before start Contributing.

- This issue is only for 'GWOC' contributors of 'Machine Learning' domain.

**To be Mentioned while taking the issue :**
- Full name
- Batch Number
- GitHub Profile Link
- Which type of Contribution you want to make : 
    - [ ] Documentation

All the best. Enjoy your open source journey ahead. 😎

### Domain

Machine Learning

### Type of Contribution

Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by monalsingh10**
<hr>

## Description 📜

I have created a Documentation on" K-Nearest -Neighbour Algorithm" .
Fixes #2811

<hr>

## Type of change 📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [ ] Documentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [ ] C/#CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [ ] Frontend Dev #HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] Java#Script
- [ ] ME#RN
- [ ] Machine Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸
image

<hr>

### CODE REVIEW

File: Machine_Learning/Supervised_Machine_Learning/ML_2811_K-Nearest-Neighbour-Algorithm_(D).md [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+
+
+# Implementing KNN Algorithm with Scikit-Learn
+
+**The Dataset:-**
+We are going to use the famous iris data set for our KNN example. The dataset consists of four attributes: sepal-width, sepal-length, petal-width and petal-length. These are the attributes of specific types of iris plant. The task is to predict the class to which these plants belong. There are three classes in the dataset: Iris-setosa, Iris-versicolor and Iris-virginica.
+
+**Importing Libraries:-**
+
+import numpy as np
```
Comment (abhisheks008) [inline Machine_Learning/Supervised_Machine_Learning/ML_2811_K-Nearest-Neighbour-Algorithm_(D).md:37]: The codes you have been written are not being distinguished, it looks like normal write up. You can add up the codes like 
``` python
import numpy as np
import matplotlib as plt
import pandas as pd
```

@monalsingh10

**Review by abhisheks008 — CHANGES_REQUESTED** [review]

File: Machine_Learning/Supervised_Machine_Learning/ML_2811_K-Nearest-Neighbour-Algorithm_(D).md (hunk 1 above)
Comment (abhisheks008) [reply Machine_Learning/Supervised_Machine_Learning/ML_2811_K-Nearest-Neighbour-Algorithm_(D).md:37]: Also the file name should not be like this. It should be like, 
`ML_2_8_K-Nearest Neighbour Algorithm_(D).md`

Comment (monalsingh10) [reply Machine_Learning/Supervised_Machine_Learning/ML_2811_K-Nearest-Neighbour-Algorithm_(D).md:37]: Okay sir I will be doing it.

Comment (monalsingh10) [reply Machine_Learning/Supervised_Machine_Learning/ML_2811_K-Nearest-Neighbour-Algorithm_(D).md:37]: @abhisheks008 Sir in document I have added libraries that is pandas , matplotlib , numpy  with proper code.

Comment (abhisheks008) [reply Machine_Learning/Supervised_Machine_Learning/ML_2811_K-Nearest-Neighbour-Algorithm_(D).md:37]: Yeah I know that, but it is not distinguished well enough. Kindly look into that.

Comment (abhisheks008) [reply Machine_Learning/Supervised_Machine_Learning/ML_2811_K-Nearest-Neighbour-Algorithm_(D).md:37]: The code snippets will be written properly.

Comment (monalsingh10) [reply Machine_Learning/Supervised_Machine_Learning/ML_2811_K-Nearest-Neighbour-Algorithm_(D).md:37]: Okay Sir!

**Comment by monalsingh10** [comment]
@abhisheks008 I did changes as per your advice! Do have a look!

**Review by abhisheks008 — CHANGES_REQUESTED** [review]
Please change the file name `ML_2_8_K-Nearest Neighbour Algorithm_(D).md`

**Comment by monalsingh10** [comment]
Okay @abhisheks008 sir

**Review by Isha307 — APPROVED** [review]

**Pull request merged**